### ISSUE TITLE
Is it possbile to eslint-disable a script(type=module)? 

### ISSUE THREAD

**Issue by wxiaoguang** [BenoitZugmeyer/eslint-plugin-html#188]
When working with backend MVC projects, sometimes the code could be:

```html
<!-- I want to disable eslint-plugin-html for this script block -->
<script type="module">
    window.myGlobal.var = {{.MyData}};  // here is invalid JS syntax, but it's rendered by backend template engine correctly.
</script>

<!--  eslint-plugin-html works well for this block, and it should be enabled -->
<script type="module">
    console.log(window.myGlobal.var);
</script>
```

What we need here is to make `eslint-plugin-html` skip a whole `<script>` block because code inside it is not valid JS syntax.

When using `eslint-plugin-html`, I can not find a proper way to disable the eslint for the script(type=module) block (I do not want ignore the whole file).

I know a trick that: `<script><!-- /* eslint-disable */ --></script>`, but it doesn't work for script(type=module)

For traditional `<script>` we can use `<script><!-- /* eslint-disable */ --></script>`.

But for `<script type=module>`, this trick doesn't work anymore because the module script doesn't accept `<!--` HTML comment.

**Comment by Standard8** [comment]
I think you should be able to do something like:

```javascript
<script type="application/javascript" type="module">
/* eslint-disable */

// Your code

/* eslint-enable */
</script>
```

**Comment by BenoitZugmeyer** [comment]
Please refer to [Linting templates](https://github.com/BenoitZugmeyer/eslint-plugin-html#linting-templates-or-php)

**Comment by silverwind** [comment]
I think the bug here is that invalid JS between `eslint-disable` and `eslint-enable` trips this module's parser when it should ideally not try to parse anything between such tags:

```html
<script>
  /* eslint-disable */
  ]
  /* eslint-enable */
</script>
```

The bug is reproducible with a regular `.js` file too, so I think `espree` is not correctly ignoring code between disable/enable sections as it expects the whole file/block to be valid JS, which generally is true for `.js` files but it's certainly a problem with templated HTML.

**Comment by BenoitZugmeyer** [comment]
Thank you for the suggestion, it has been implemented and released as part of v7.0.0.
